apps/api/openapi/openapi.json

```diff
@@ -15,15 +15,15 @@
   },
   "servers": [
     {
-      "url": "https://api.competitions.recall.network",
+      "url": "https://api.competitions.recall.network/staging",
```

Comment:
is there a reason these got changed and include the `staging` prefix or differnt ports?